docker/package/tezos_baking_wizard.py

```diff
@@ -0,0 +1,489 @@
+                print("Exiting Tezos Baking Wizard")
+                return None
+            elif answer.lower() in ["help", "?"]:
+                print(step.help)
+            else:
+                if not answer and step.default is not None:
+                    answer = step.default
+
+                try:
+                    answer = step.validator.validate(answer)
+                except ValueError as e:
+                    print(color("Validation error: " + str(e), "\x1b[1;31m"))
+                else:
+                    validated = True
+                    values[step.id] = answer
+
+    return values
+
+
+class Setup:
+    def __init__(self, config):
+        self.config = config
+
+    # Check if an account with the 'baker' alias already exists, and ask the user
+    # if it can be overwritten.
+    def check_baker_account(self, cmd):
+        output = subprocess.run(shlex.split(cmd), capture_output=True)
+        if output.returncode and re.search(
+            b"The secret_key alias baker already exists.", output.stderr
+        ):
+            address = re.search(b"The current value is (\w+:\w+)", output.stderr).group(
+                1
+            )
+            print("An account with the 'baker' alias already exists.")
+            print("Its address is ", address.decode("utf-8"))
+
+            replace = input("Should it be replaced with the new import? <y/N> ").strip()
+            if replace.lower() in ["y", "yes"]:
+                proc_call(cmd + " --force")
+
+    # Check if there is already some blockchain data in the tezos-node data directory,
+    # and ask the user if it can be overwritten.
+    def check_blockchain_data(self, snapshot_file):
+        node_dir = "/var/lib/tezos/node-" + self.config["network"]
+        node_dir_contents = os.listdir(node_dir)
+        clean = ["config.json", "version.json"]
+        diff = set(node_dir_contents) - set(clean)
+        if diff:
+            print("The Tezos node data directory already has some blockchain data:")
+            print("\n".join(diff))
+            print()
+            replace = input(
+                "Delete this data and bootstrap the node again? <y/N> "
+            ).strip()
+            if replace.lower() in ["y", "yes"]:
+                for path in diff:
+                    if proc_call("sudo rm -r " + os.path.join(node_dir, path)):
+                        print(
+                            "Could not clean the Tezos node data directory. "
+                            "Please do so manually."
+                        )
+                        raise OSError(
+                            "'sudo rm -r " + os.path.join(node_dir, path) + "' failed."
+                        )
+
+                print("Node directory cleaned.")
+
+                return True
+
+        return False
+
+    # Importing the snapshot for Node bootstrapping
+    def import_snapshot(self):
+        do_import = self.check_blockchain_data(snapshot_file)
```

Comment:
:thinking: currently I get an error here, due to the fact that `snapshot_file` is undefined.